enigma-swing/src/main/java/org/quiltmc/enigma/gui/config/Config.java

```diff
@@ -69,20 +94,25 @@ public final class Config extends ReflectiveConfig {
 	public final DevSection development = new DevSection();
 
 	/**
-	 * The look and feel stored in the config: do not use this unless setting! Use {@link #activeLookAndFeel} instead,
+	 * The look and feel stored in the config: do not use this unless setting! Use {@link #activeThemeProperties} instead,
 	 * since look and feel is final once loaded.
 	 */
-	public final TrackedValue<LookAndFeel> lookAndFeel = this.value(LookAndFeel.DEFAULT);
+	public final TrackedValue<ThemeProperties> theme = this.value(ThemeProperties.DEFAULT);
```

Comment:
this can be annotated with @Alias to prevent breaking configs

Reply:
oh cool, I'll add one for `colors`->`syntax_pane_colors`, too

Reply:
well, it won't be possible to migrate colors if we go through with moving them to individual files

Reply:
yeah, I hadn't seen your separate files comment yet
I've finished the separating files, updating the PR description now